recipe/meta.yaml

```diff
@@ -13,10 +13,11 @@ source:
     - fix_dispatch_apply_auto.patch
 
 build:
-  number: 0
+  number: 1
   run_exports:
     - {{ pin_subpackage('pytorch', max_pin='x.x.x') }}
   skip: true  # [win]
+  skip: true  # [cuda_compiler_version in ("10.2", "11.0", "11.1")]
```

Comment:
Could this be done more appropraitely if you simply put `cuda_compiler_version = 11.2` in the build config yaml?